rules/library.bzl

```diff
@@ -675,17 +722,6 @@ def apple_library(name, library_tools = {}, export_private_headers = True, names
         import_vfsoverlays.append(import_name + "_vfs")
 
     for xcframework in kwargs.pop("vendored_xcframeworks", []):
-        import_headers = native.glob(
-            ["**/*.h"],
-        )
-        import_module_maps = native.glob(
-            ["**/*.modulemap"],
-        )
-        if len(import_module_maps) > 0:
-            import_module_map = import_module_maps[0]
-        else:
-            import_module_map = None
-        vfs_imported_framework = _find_imported_framework_name(import_headers)
```

Comment:
This code doesn't do anything since `vfs_imported_framework` isn't used here, instead this is created for each slice in the `_xcframework_slice` function.

Reply:
Makes sense, I added a `fail` if `slices` is empty!